packages/enzyme-matchers/src/assertions/toMatchElement.js

```diff
@@ -25,7 +25,7 @@ function toMatchElement(
 
   const actual = actualEnzymeWrapper.debug(options);
   const expected = expectedWrapper.debug(options);
-  const pass = actual === expected;
+  const pass = actual.equals(expected);
```

Comment:
This also won't work because `actual` is a string. You are wanting to do `actualEnzymeWrapper.equals(expectedWrapper)`